packages/ui/src/components/va-form/VaForm.stories.ts

```diff
@@ -1,12 +1,286 @@
+  })
+
+  await step('Does not display error message', async () => {
+    expect(secondInput.getAttribute('aria-invalid')).toEqual('false')
+  })
+}
+
+export const HideErrorMessages = () => ({
+  components: { VaForm, VaInput },
+  template: `
+    [true]
+    <va-form
+      role
+      immediate
+      hideErrorMessages
+    >
+      <va-input :rules="[value => !!value || 'input is required']"/>
+    </va-form>
+    [false]
+    <va-form
+      role
+      immediate
+      :hideErrorMessages="false"
+    >
+      <va-input :rules="[value => !!value || 'input is required']"/>
+    </va-form>
+  `,
+})
+
+HideErrorMessages.play = async ({ step }) => {
+  const [firstDefinedInput, secondDefineInput] = document.querySelectorAll('[role] > div > div:nth-child(2)')
+
+  await step('Does not display validation error', async () => {
+    expect(firstDefinedInput).toBeDefined()
+  })
+
+  await step('Displays validation error', async () => {
+    expect(secondDefineInput).toBeUndefined()
+  })
+}
+
+export const HideErrors = () => ({
+  components: { VaForm, VaInput },
+  template: `
+    [true]
+    <va-form
+      role
+      immediate
+      hideErrors
+    >
+      <va-input :rules="[value => !!value || 'input is required']"/>
+    </va-form>
+    [false]
+    <va-form
+      role
+      immediate
+      :hideErrors="false"
+    >
+      <va-input :rules="[value => !!value || 'input is required']"/>
+    </va-form>
+  `,
+})
+
+HideErrors.play = async ({ step }) => {
+  const [firstInput, secondInput] = document.querySelectorAll('[role] > div')
+
+  await step('Does not highlight invalid input', async () => {
+    expect(firstInput).not.toHaveClass('va-input-wrapper--error')
+  })
+
+  await step('Highlights invalid input', async () => {
+    expect(secondInput).toHaveClass('va-input-wrapper--error')
+  })
+}
+
+export const Focus = () => ({
+  components: { VaForm, VaInput, VaButton },
+  template: `
+    <va-form ref="form">
+      <va-input />
+    </va-form>
+    <va-button @click="$refs.form.focus()">
+      Focus first input
+    </va-button>
+  `,
+})
+
+Focus.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const firstInput = canvas.getByRole('textbox', { name: '' }) as HTMLElement
+  const button = canvas.getByRole('button', { name: 'Focus first input' }) as HTMLElement
+
+  await step('Focusses first input', async () => {
+    await userEvent.click(button)
+    expect(firstInput).toHaveFocus()
+  })
+}
+
+export const FocusInvalid = () => ({
+  components: { VaForm, VaInput, VaButton },
+  template: `
+    <va-form ref="form">
+      <va-input error />
+    </va-form>
+    <va-button @click="$refs.form.focusInvalidField()">
+      Focus invalid
+    </va-button>
+  `,
+})
+
+FocusInvalid.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const input = canvas.getByRole('textbox', { name: '' }) as HTMLElement
```

Comment:
Maybe you can think of some shortcuts for selectors like this?

I'm thinking something like:

```
const input = helper.get('input')
const button = helper.get('button')
```

or maybe even
```
const [input, button] = helper.get('input', 'button')
```

There is only one button and one input in story, so we can definitely add some sugar.